RenderSystems/GLSupport/src/EGL/Android/OgreAndroidEGLWindow.cpp

```diff
@@ -119,7 +119,9 @@ namespace Ogre {
                                 "currentGLContext was specified with no current GL context",
                                 "EGLWindow::create");
                 }
-
+#if defined(OGRE_BITES_HAVE_SDL) && OGRE_BITES_HAVE_SDL
```

Comment:
do not use defines here, but rather pass `currentGLSurface=true` via miscParams. See the `currentGLContext` code just above.
You can use #ifdef in OgreBites to conditionally pass this flag.

Reply:
@paroj 
where should I put `if (currentGLSurface=true)`?
is inside the `if (currentGLContext = true)` above accepable?